lib/graphql/language/lexer.rb

```diff
@@ -1415,10 +1415,10 @@ def self.emit(token_name, ts, te, meta)
 
 def self.emit_string(ts, te, meta, block:)
 quotes_length = block ? 3 : 1
-ts += quotes_length
-value = meta[:data][ts...te - quotes_length].pack(PACK_DIRECTIVE).force_encoding(UTF_8_ENCODING)
+content_range = (ts + quotes_length)...(te - quotes_length)
+value = meta[:data][content_range].pack(PACK_DIRECTIVE).force_encoding(UTF_8_ENCODING) || ''
```

Comment:
This is assuming that having no value means we have an empty string. Not sure if it is the best approach.